pilot/pkg/model/config.go

```diff
@@ -829,6 +853,56 @@ func (store *istioConfigStore) QuotaSpecByDestination(instance *ServiceInstance)
 	return out
 }
 
+func (store *istioConfigStore) AuthenticationPolicyByDestination(hostname string, port *Port) *authn.Policy {
```

Comment:
comment needs for public method

Reply:
It's there in the interface (few lines above). I can copy it here but it seems not the convention in this file.

Reply:
oh. never mind.